### ISSUE TITLE
Onboard Ilya

### ISSUE THREAD

**Issue by dianagriffin** [department-of-veterans-affairs/abd-vro#1824]
Welcome, Ilya! We are excited to have you on the team! 

This ticket exists to lay out some logistical to-do's alongside areas of focus for getting up to speed on the VRO team and our work. 

### To be done for you - @dianagriffin / @smythe5
- [x] Invite to DSVA, Lighthouse Slack spaces
  - [x] Add to slack groups (@vro-team)
  - [x] Introduce in #benefits-onboarding channel (to kick off OCTO onboarding steps) and #benefits-vro team channel
- [x] Add to recurring team meeting invites
- [x] Invite to the VA ABD Team Calendar in Google
- [x] Invite to the OCTO Benefits Delivery Team room in the VA Mural Workspace (as a guest)
- [x] Add to VA Github Org ([form here](https://github.com/department-of-veterans-affairs/github-user-requests/issues/new/choose)) - _pending clearance paperwork_
  - [x] Add to [abd-vro Github team](https://github.com/orgs/department-of-veterans-affairs/teams/va-abd-rrd/members?add=true)
- [x] Add to the VA ABD Sharepoint site as an Owner (pending VA email)
- [x] Request an Okta account (https://jira.devops.va.gov/servicedesk/customer/portal/1/group/252?groupId=252)
  - Request template: _Name is a new member of the OCTO Benefits VRO team. Requesting an Okta account be created for First Last (include VA email) with the following applications assigned:_
     - _VA Jira_
     - _VA Confluence_
     - _lighthousedi-va-datadog, with "Standard Role" permissions_

###  To do - Ilya

- [ ] Join relevant [slack channels](https://docs.google.com/spreadsheets/d/1c3e6SVuhVz3q4bh1qf5jj25EmusCd3NVXhDjSIpr9yk/edit#gid=1020923043) -- you don't have to join ALL these channels; start with the #benefits-vro-* and #benefits-cft-* and you can expand from there as you get more involved in the work. Feel free to ask the team in #benefits-vro if you're wondering about whether to join a particular channel.
   - Most relevant channels for you: #benefits-vro (our team channel for coordination across the VRO team), #benefits-vro-engineering (engineering-specific collaboration for the VRO team), #benefits-vro-support (channel for collaboration with VRO partner teams), #benefits-cft (cross-team collaboration channel for the 3 teams in the Claims Fast-Tracking Crew)
- [x] Watch ["The Way We Work" presentation](https://us06web.zoom.us/rec/share/sHRSblAbF229tQpj7-Nqmmdi7Zb5bWclCd0lqMMdGAttuaJZDByZy8C7vi1BO9Mo.7XuVLn108mwIpnQj?startTime=1680030182000)
- [x] Go through Benefits Portfolio onboarding materials and onboarding buddy meetings (see #benefits-onboarding channel in Slack; you'll be tagged there with your onboarding buddy assignments and materials to review)
- [x] Get [added to the VA GitHub organization](https://github.com/department-of-veterans-affairs/github-user-requests/issues/new/choose) (ask someone who is already in the org)
- [x] Get added to the [OCTO-VRO GitHub team](https://github.com/orgs/department-of-veterans-affairs/teams/octo-vro/members) to have access to VRO's GitHub repos
- [x] If you need access to VRO's LHDI deployment environments, get added to the [VA-ABD-RRD GitHub team](https://github.com/orgs/department-of-veterans-affairs/teams/va-abd-rrd/members)
When you have your VA.gov email, connect it to your GitHub account by adding as an email in your settings. You must also make sure the name on your profile matches your name. This will allow you to perform SSO to various apps through GitHub on the VA's network, e.g. Aqua.
- [ ] Add your name and skills to [POCs](https://github.com/department-of-veterans-affairs/abd-vro/wiki/POCs)
- [ ] Read VRO's [wiki](https://github.com/department-of-veterans-affairs/abd-vro/wiki#introduction), in particular the following (some content may need updating):
   - [ ] [Problems being solved](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Problems-being-solved) and  [Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) provides context
   - [ ] [Routing-API-requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Routing-API-requests), [Code structure](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Code-structure), and [Development process](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Development-process) (from [Slack thread](https://dsva.slack.com/archives/C031RRFCHV1/p1662673096036869))
   - [ ] (Optional) If interested in the initial incarnation of VRO (before Amida touched it), watch old recording '2022-06 VRO code walk-through.mp4' in [Shared Meeting Notes](https://dvagov.sharepoint.com/:f:/r/sites/vaabdvro/Shared%20Documents/VRO%20-%20Virtual%20Regional%20Office/Shared%20Meeting%20Notes?csf=1&web=1&e=0LcwED)
   - [ ] [Gradle projects](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Code-structure#gradle-projects)
   - [ ] [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) is a good intro to the codebase history
   - [ ] [LHDI's Boilerplate Instructions](https://github.com/department-of-veterans-affairs/abd-vro/wiki/LHDI%27s-Boilerplate-Instructions) and if needed [Setup on Windows](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Setup-on-Windows). Amida wrote up some setup instructions for Windows in [Software Design Document](https://docs.google.com/document/d/1WN9bRWgrV08UhdefSmKxt7hQu5OaVKhD8TwrqkofEsQ/edit) (linked from [Roadmap#references](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap#references))
   - [ ] [External APIs](https://github.com/department-of-veterans-affairs/abd-vro/wiki/External-APIs) to interact with other systems
   - [ ] Review the materials on [this wiki page](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Amida-Hand-off-Resources) for walkthroughs of the code
- [ ] Watch [this demo](https://lighthouseva.slack.com/archives/C03UA9MV1EH/p1681327320119849?thread_ts=1681304404.488959&cid=C03UA9MV1EH) from the Lighthouse Delivery Infrastructure (LHDI) team on the SecRel pipeline and SD Elements
- [ ] Remember to periodically add what you're working on (or responsible for) to the [Engineering POCs spreadsheet](https://docs.google.com/spreadsheets/d/1R0fmGgKubLbmaX-KpeO0Fm5Sx7DayrnMcwZAUcj2F_M/edit#gid=0) (pinned at the top of the #benefits-rrd-engineering Slack channel)
- [ ] For background, context, and motivation, read [VA Automated Benefits Delivery Team - Team Onboarding](https://docs.google.com/document/d/1SnJOqDtlb2UkO9DNczZHCVTTQURiCvp81hBJhrlv6Xs/edit?usp=sharing) as a lower priority (whenever you have downtime between meetings or waiting on blockers to be resolved). Important topics from that document include:
   - [ ] [Teams and vision](https://dvagov.sharepoint.com/:p:/r/sites/HypertensionRapidDecisionPilot/_layouts/15/Doc.aspx?sourcedoc=%7B4A6FD69B-D04E-4CD8-879A-AE2C547B7A66%7D&file=ABD%20Vision%20Snapshot%20_%20May%202022.pptx&action=edit&mobileredirect=true): various responsibilities of various teams (requires sharepoint)
   - [ ] [Slack](https://docs.google.com/document/d/1GUqhDxxJUdT51u-73PRjls2LqqrfYNW9PPkpaWChwvc/edit?usp=sharing#heading=h.2324h4tbjddu) usage

**Comment by dianagriffin** [comment]
Okta account requested: https://jira.devops.va.gov/servicedesk/customer/portal/1/DOTSD-30319
